mesatee_services/fns/sgx_trusted_lib/Cargo.toml

```diff
@@ -16,6 +16,7 @@ mesalock_sgx = ["sgx_tstd", "mesatee_core/mesalock_sgx", "fns_proto/mesalock_sgx
 cov = ["sgx_cov"]
 
 [dependencies]
+prost = "*"
```

Comment:
version?

Reply:
FNS doesn't need it at this time. I will remove it.